src/wheatley.ts

```diff
@@ -331,6 +335,30 @@ export class Wheatley {
         this.setup(config).catch(this.critical_error.bind(this));
     }
 
+    private *locate_components(config: core_config) {
+        const visited = config.components?.include ? new Set<string>() : undefined;
+
+        const pw = new PathScurry(import.meta.dirname);
+
+        for (const file of globIterateSync("**/components/**/*.js", {
+            ignore: config.components?.exclude,
+            scurry: pw,
+            withFileTypes: true,
+        })) {
+            yield file.relativePosix();
+            visited?.add(file.fullpath());
+        }
+
+        if (config.components?.include) {
+            for (const file of config.components.include) {
+                const path = pw.resolve(file);
+                if (!visited!.has(path)) {
+                    yield file;
```

Comment:
Above uses `relativePosix`, should something similar be used here?

Reply:
I believe doing that would be redundant since `file` is already interpreted as relative to the working directory of `pw`, so this would just undo the `resolve()` we just did?